telemetry.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+package openmind;
+
+option csharp_namespace = "OpenMind";
+
+service TelemetryService {
+  /**
+   * TODO (BNR): We need endpoints for:
+   *             1. Version number - should respond with the server version
+   *             2. Supported devices - should respond with "Medtronic Summit RC+S"
+   *             3. Inspect Repository - should respond with the cached bridges/devices
+   */
+
+  /**
+  * List supported devices.
+  */
+  rpc SupportedDevices (SupportedDevicesRequest) returns (SupportedDevicesResponse);
+  /**
+   * Inspect Repository.
+   */
+  rpc InspectRepository (InspectRepositoryRequest) returns (InspectRepositoryResponse);
+}
+
+message SupportedDevicesRequest {
+  /**
+   * Scheme-less URI describing the bridge resource.
+   * For example, '//summit/bridge/xxxxxxxxx' where
+   * 'xxxxxxxxx' is the serial number of the bridge.
+   */
+  string name = 1;  
+}
+
+message SupportedDevicesResponse {
+  /**
+   * String with supported devices.
+   */
+  string supported_devices = 1;
```

Comment:
This field should be repeated in case we support multiple devices in one of these microservices